disk_usage/disk_usage.py

```diff
@@ -27,40 +28,33 @@
 
 
 def aggregate_level(name: str) -> str:
-    """Returns a prefix for the given blob name at the aggregation level."""
+    """Returns a prefix for the given blob name at the folder or Hail table level."""
     ht_index = name.find('.ht/')
     if ht_index != -1:
         return name[: ht_index + 3]
     mt_index = name.find('.mt/')
     if mt_index != -1:
         return name[: mt_index + 3]
```

Comment:
You could probably use the [walrus syntax](https://docs.python.org/3/whatsnew/3.8.html#assignment-expressions) here:

```suggestion
    if (ht_index := name.find('.ht/')) != -1:
        return name[: ht_index + 3]
    if (mt_index := name.find('.mt/')) != -1:
        return name[: mt_index + 3]
```

Reply:
Nice, thanks!